pkg/pdatautil/attributes.go

```diff
@@ -0,0 +1,41 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package pdatautil // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatautil"
```

Comment:
This means that this package is part of the public API for contrib. Shouldn't it be under an internal module? Like here:

https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/internal

Reply:
This is where I'd like to hear from reviewers, and since there's no other place to reference this function right now, I think it makes more sense to put it inside internal